kickstarts/base.ks.erb

```diff
@@ -3,6 +3,7 @@
 logging --level=debug
 
 #version=DEVEL
+text
```

Comment:
This is super helpful in debugging, I know we had it at one time because @jrafanie ran into an error and this was the only way to see what it was.